website/docs/r/ecr_registry_policy.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+subcategory: "ECR"
+layout: "aws"
+page_title: "AWS: aws_ecr_registry_policy"
+description: |-
+  Provides an Elastic Container Registry Policy.
+---
+
+# Resource: aws_ecr_registry_policy
+
+Provides an Elastic Container Registry Policy.
+
+## Example Usage
+
+```hcl
+data "aws_caller_identity" "current" {}
+
+data "aws_region" "current" {}
+
+data "aws_partition" "current" {}
+
+resource "aws_ecr_registry_policy" "example" {
+  policy = <<EOF
+{
+    "Version":"2012-10-17",
+    "Statement": [
+        {
+            "Sid": "testpolicy",
+            "Effect": "Allow",
+			"Principal":{
+                "AWS":"arn:${data.aws_partition.current.partition}:iam::${data.aws_caller_identity.current.account_id}:root"
+            },
+            "Action": [
+                "ecr:ReplicateImage"
+			],
+			"Resource": [
+                "arn:${data.aws_partition.current.partition}:ecr:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:repository/*"
+            ]
+        }
+    ]
+}
+EOF
```

Comment:
Use 'jsonencode` style instead of heredoc:
https://github.com/hashicorp/terraform-provider-aws/pull/16792.